src/Illuminate/Database/Eloquent/Concerns/HasCasts.php

```diff
@@ -341,11 +327,10 @@ protected function toCustomCastable($key, $value = null)
     }
 
     /**
-     * @param string $key
+     * @param  string  $key
+     * @return Castable
```

Comment:
Needs to be fully qualified.

Reply:
Sorry, I don’t understand what needs to be done.
The codestyle led to the look specified in the documentation: https://laravel.com/docs/6.x/contributions#phpdoc

Reply:
Correctly understand that I need to add a description of the methods?

Reply:
You need to use the fully qualified class name: `\Illuminate\Contracts\Database\Eloquent\Castable`, like the example you posted there, where we have `\Closure|string|null`.